retis/src/collect/collect.rs

```diff
@@ -317,17 +317,37 @@ impl Collectors {
         }
         Self::setup_filters(self.probes.builder_mut()?, collect)?;
 
-        // If probe_stack is on and user hasn't provided a starting point, use
-        // skb:consume_skb & skb:kfree_skb.
-        if collect.probe_stack && collect.probes.is_empty() {
-            self.probes
-                .builder_mut()?
-                .register_probe(Probe::raw_tracepoint(Symbol::from_name(
-                    "skb:consume_skb",
-                )?)?)?;
-            self.probes
-                .builder_mut()?
-                .register_probe(Probe::raw_tracepoint(Symbol::from_name("skb:kfree_skb")?)?)?;
+        // If auto_mode is on and no probe is given, find the right set
+        // automagically.
+        if self.auto_mode && collect.probes.is_empty() {
+            let mut probes = if collect.probe_stack {
+                // If --probe-stack is used, use skb:consume_skb & skb:kfree_skb
+                // as a starting point (these should capture most if not all of
+                // the packets and help moving up the stack).
+                vec![
+                    Probe::raw_tracepoint(Symbol::from_name("skb:consume_skb")?)?,
```

Comment:
This is triggered *very* often for non-critical packets. Isn't this going to add a huge amount of noise?

Reply:
(This is not introduced by this PR)

That is if `--probe-stack` is set and no probe is given. Then the idea is to match (a filter is required for `--probe-stack`) packets and rewind their journey. We definitively want to catch non-drop traffic here.